_config.yml

```diff
@@ -10,9 +10,9 @@ title: "Stitch Fix Technology"
 production_url: "http://technology.stitchfix.com"
 atom_path: "atom.xml"
 
-# Your UA tracking number (i.e 35858753-1) for Google Analytics. 
-# If you comment this out, or don't fill out this parameter - Google analytics won't be included. 
-googleID: 25598599-8
+# Your UA tracking number (i.e 35858753-1) for Google Analytics.
+# If you comment this out, or don't fill out this parameter - Google analytics won't be included.
+# googleID: 25598599-8
```

Comment:
This wasn't actually tracking anything so it must be left over from Adam's version and used some plugin that doesn't exist in the repo anymore.